src/pages/EditSplitBillPage.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import CONST from '../CONST';
+import ROUTES from '../ROUTES';
+import ONYXKEYS from '../ONYXKEYS';
+import compose from '../libs/compose';
+import transactionPropTypes from '../components/transactionPropTypes';
+import * as ReportUtils from '../libs/ReportUtils';
+import * as IOU from '../libs/actions/IOU';
+import * as CurrencyUtils from '../libs/CurrencyUtils';
+import Navigation from '../libs/Navigation/Navigation';
+import FullPageNotFoundView from '../components/BlockingViews/FullPageNotFoundView';
+import EditRequestDescriptionPage from './EditRequestDescriptionPage';
+import EditRequestMerchantPage from './EditRequestMerchantPage';
+import EditRequestCreatedPage from './EditRequestCreatedPage';
+import EditRequestAmountPage from './EditRequestAmountPage';
+
+const propTypes = {
+    /** Route from navigation */
+    route: PropTypes.shape({
+        /** Params from the route */
+        params: PropTypes.shape({
+            /** The transaction field we are editing */
+            field: PropTypes.string,
+
+            /** The chat reportID of the split */
+            reportID: PropTypes.string,
+
+            /** reportActionID of the split action */
+            reportActionID: PropTypes.string,
+        }),
+    }).isRequired,
+
+    /** The current transaction */
+    transaction: transactionPropTypes.isRequired,
+
+    /** The draft transaction that holds data to be persisted on the current transaction */
+    draftTransaction: PropTypes.shape(transactionPropTypes),
+};
+
+const defaultProps = {
+    draftTransaction: {},
+};
+
+function EditSplitBillPage({route, transaction, draftTransaction}) {
+    const fieldToEdit = lodashGet(route, ['params', 'field'], '');
+    const reportID = lodashGet(route, ['params', 'reportID'], '');
+    const reportActionID = lodashGet(route, ['params', 'reportActionID'], '');
+
+    const {
+        amount: transactionAmount,
+        currency: transactionCurrency,
+        comment: transactionDescription,
+        merchant: transactionMerchant,
+        created: transactionCreated,
+    } = draftTransaction ? ReportUtils.getTransactionDetails(draftTransaction) : ReportUtils.getTransactionDetails(transaction);
```

Comment:
This will always return true because we add a default `{}`, no?

Reply:
Will remove the default as the draft shouldn't be required

Reply:
Will make it undefined in the follow up PR